js/index.js

```diff
@@ -1 +1,101 @@
 // Your code goes here
+const title = document.querySelector('.logo-heading');
+
+title.addEventListener('mouseover',function() {
+    title.style.color = '#FDBA58';    //@bus-yellow
+})
+
+title.addEventListener('mouseout',function() {
+    title.style.color = 'black';
+})
+
+const h2 = document.querySelectorAll('h2');
+
+h2.forEach( function(h2tag) {
+    h2tag.addEventListener('mouseover', () => h2tag.style.color = '#FDBA58' );
+})
+
+h2.forEach( h2tag => h2tag.addEventListener('mouseout', () => h2tag.style.color = 'black'));
+
+const h4 = document.querySelectorAll('h4');
+
+h4.forEach( h4tag => h4tag.addEventListener('mouseover', () => h4tag.style.color = '#FDBA58'));
+
+h4.forEach(h4tag => h4tag.addEventListener('mouseout', () => h4tag.style.color = 'black'));
+
+
+const a = document.querySelectorAll('.nav-link');
+
+a.forEach(atag => atag.addEventListener('mouseover', function() {
+    atag.style.color = '#FDBA58';
+}));
+
+a.forEach(atag => atag.addEventListener('mouseout', function() {
+    atag.style.color = 'black';
+}));
+
+const input = document.querySelector('#input');
+
+input.addEventListener('focus', function() {
+    input.style.backgroundColor = '#FDBA58';
+    input.style.color = 'white';
+    if (input.classList.contains('invalid')) {
+        input.classList.remove('invalid');
+        error.textContent = '';
+    }
+})
+
+const error = document.querySelector('#error');
+
+input.addEventListener('blur', function() {
+    input.style.background = 'white';
+    input.style.color = 'black';
+    if(!input.value.includes('@')) {
+       input.classList.add('invalid');
+       error.textContent = 'Please enter a correct email';
+       error.style.color = 'red';
+       }
+});
+
+const busImg = document.querySelector('img');
+
+busImg.addEventListener('mouseover', () => {
+    busImg.style.borderRadius = '100%';
+    busImg.style.transition = 'all 2s';
+    busImg.style.objectFit = 'cover';
+    //can't figure out how to crop the image on left and right.  tried doing the object-fit : cover
+//    busImg.style.height = '100';
+//    busImg.style.width: '100';   //and why do these properties suspend the animation?  
```

Comment:
You could try adding width and height to the CSS file and see what happens then. Also be sure to add units.